his/core/application-integration-ladesigner-1.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Logic Apps Integration | Microsoft Docs"
```

Comment:
I'm tweaking the search results title here to prevent people thinking this doc is about Azure Logic Apps itself, land on the page and then "bounce" (leaving within 5 seconds). This behavior is bad for content performance.

```suggestion
title: Extend Host Integration Server with Azure Logic Apps
```